packages/pn-personafisica-webapp/src/redux/sidemenu/actions.ts

```diff
@@ -21,7 +21,15 @@ export const getSidemenuInformation = createAsyncThunk<Array<Delegator>>(
   }
 );
 
+// PN-7095 - per capire quale categorie di recapito far vedere nel DomicileBanner
+// si devono prendere gli indirizzi default, tranne per AppIO che si prendono tutti (senza verificare default) 
+// il cui valore non sia DISABLED (cfr. description e commenti della issue JIRA)
 export const getDomicileInfo = createAsyncThunk<Array<DigitalAddress>>(
   'getDomicileInfo',
-  performThunkAction(async () => (await ContactsApi.getDigitalAddresses()).legal)
-);
+  performThunkAction(async () => {
+    const isDefaultAddress = (address: DigitalAddress) => 
+      address.senderId === 'default' || (address.channelType === CourtesyChannelType.IOMSG && address.value !== IOAllowedValues.DISABLED);
```

Comment:
Whit this condition also senderId = 'default' && channelType = IOMSG && value = DISABLED passes

Reply:
omg you're right